### ISSUE TITLE
Github issue Integration made right

### ISSUE THREAD

**Issue by tonylampada** [freedomsponsors/www.freedomsponsors.org#254]
Github Integration currently works like this:
- User connects Github account to FreedomSponsors
- On freedomsponsors.org, user selects repositories that he wants to integrate
- FreedomSponsors watches those repos, and adds a comment on every new issue that is open with a "sponsor this issue" link, pointing back to FreedomSponsors.

This is not as good as I thought initially.
Every new comment triggers an email from Github, not just for the repo owner, but all the repo watchers too.
When there are a lot of issues, it gets very annoying.

So much that @tduchateau opened #253 (sorry about that! ^^)
I'm betting there are more people annoyed by that besides @tduchateau, so I decided to turn off the commenter bot as soon as I saw #253.
This means that Github Integration feature is temporarily broken, until we reimplement it "the right way", which is:
- User connects Github account to FreedomSponsors
- On freedomsponsors.org, user chooses to enable Github Integration
- freedomsponsors.org will ask user for more permissions to do stuff on Github (edit issues) on his behalf.
- User authorizes freedomsponsors on Github and proceeds to select repositories that he wants to integrate
- Freedomsponsors will use the github API to edit the issue description (with the user's github credentials), adding the "sponsor this issue"  link. So the link will be part of the issue description instead of a new comment.

If the user wants to disable the integration, he can do so either on freedomsponsors by deselecting the repository, or on Github by unauthorizing FreedomSponsors.org.

Thank you @tduchateau for pointing this problem out!

**Comment by knocte** [comment]
Hi @tonylampada I'm interested in sponsoring someone to work on a gitlab (not github) integration patch. Would you be open to review/mentor his contribution? I figured to post here since the github integration has been disabled for so long, but I'm not interested in sponsoring its resurrection, but an alternative platform (gitlab being more "opensourcy" than github).

**Comment by tonylampada** [comment]
Yes, Absolutely. Right now I'm too busy to work on it myself but I can definitely point someone else in the right direction to get it done.
